### PR TITLE
SEO-187952 ASPNET-Web-Forms-SEO-187952-ASPNET Web Forms HTTP to HTTPS

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan,

Please review the below PR,

Title | Description
-- | --
Task Category | HTTP to HTTPS|
Content Task Link/Mail Screenshot | NA|
|Hotfix |https://github.com/syncfusion-content/asp.netwebforms-docs/pull/541|
Ticket/Task link |https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187952|

|Changes Made | Reason for change |
|-- | -- |
|Changed HTTP links to HTTPS | To avoid redirect links | 
|Modified H1 | to enhance user experience |
|Modified description | to improve user experience |

### CODE REVIEW

File: aspnet/RichTextEditor/Working-with-Content.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Working with content related operation|RTE|Syncfusion|ASP.NET
-description: Working with Content related changes in RichTextEditor control
+description: Learn here about working with content related changes in ASP.NET Web Forms RichTextEditor control, its,elements, and more.
```
Comment (GowthamanMayil) [inline aspnet/RichTextEditor/Working-with-Content.md:4]: Check this
image

**Review by GowthamanMayil — COMMENTED** [review]
Commented

File: aspnet/RichTextEditor/Working-with-Content.md (hunk 1 above)
Comment (Yvone-Atieno) [reply aspnet/RichTextEditor/Working-with-Content.md:4]: changed

**Review by Girija-Rajendran — APPROVED** [review]
fine

**Pull request merged**